specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/ManagementPartner.json

```diff
@@ -291,7 +291,8 @@
           "description": "This is the object id."
         },
         "version": {
```

Comment:
You must update the example JSON file to change the version from string "3" to int 3 in order for the validation check to pass

Reply:
@allenjzhang , Thanks for your suggestion. I've fixed it.